README.md

```diff
@@ -10,6 +10,7 @@ So far it works like this:
 
 ```bash
 TERASLICE_URL="https://localhost" \
+TERASLICE_DISPLAY_URL="https://teraslice-xyz.lan" \
```

Comment:
can you indent this like the following lines to help emphasize the continuation here ... and the other example below.